packages/pluggableWidgets/maps-native/src/Maps.tsx

```diff
@@ -245,6 +249,10 @@ export class Maps extends Component<Props, State> {
     }
 }
 
+function isValidCoordinate(value: Big | number | undefined): boolean {
```

Comment:
If you'd like, you could import `Option` from `mendix` module, then write like `Option<Big | numer>`. If you don't want to change it, it's good to know for next time :)